ext/ext.md

```diff
@@ -12,13 +12,13 @@ extensions.
 Registry level extensions may be discovered with a standard GET as follows.
 
 ```HTTP
-    GET /v2/_ext/
+    GET /v2/_ext/discover
```

Comment:
I looked over the linked discussion, but I'm still not understanding why to make the discovery into an endpoint rather than just at the base `/v2/_ext/`?